charter.txt

```diff
@@ -15,10 +15,11 @@ many extensions into HTTP. The group will limit itself to supporting the
 functionality in widespread actual usage today.
 
 The protocol uses HTTP techniques for authentication and authorization
-(notably OAuth), and requires hop by hop encryption (i.e., https). The
-protocol will also allow for e2e media encryption, although keying is
-out of scope, and is expected to be handled by other protocols such as
-MLS. This extension will also utilize STIR for callerID.
+and requires hop by hop encryption (i.e., https). The protocol will
+also specify a technnique for encapsulated e2e encrypted media
```

Comment:
I would propose that E2E encryption of media should be mandatory, just like SRTP is mandatory in WebRTC.  It should never be possible for elements in the middle just to grab stuff, without explicit authorization.  So something like:

> The protocol will carry media in an end-to-end encrypted form in order to prevent access by unauthorized intermediary devices.  (Some authorized intermediaries may be provided with keys, depending on the key management protocol.)